thoth/glyph/cli.py

```diff
@@ -39,7 +39,7 @@
 
 _LOGGER = logging.getLogger("glyph")
 
-DEFAULT_MODEL_PATH = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), "data/model_commits_v2_quant.bin")
+DEFAULT_MODEL_PATH = "data/model_commits_v2_quant.bin"
```

Comment:
This will not work as the package will be installed. Model is not relative to PWD of thoth-glyph process but relative to thoth-glyph files.

You can try installing glyph into a virtual environment to check if it works fine. Something like this should work:

```
mkdir -p /tmp/test-glyph
cd /tmp/test-glyph
python3 -m venv venv
source venv/bin/activate
pip3 install /path/to/git/repo
# or python3 /path/to/git/repo/setup.py install
```

Reply:
You can also try behavior with test.pypi.org.

Reply:
> This will not work as the package will be installed. Model is not relative to PWD of thoth-glyph process but relative to thoth-glyph files.

Oh, I didn't know that.
Fixed. Now referring the path relative to the file.